app/models/line_item.rb

```diff
@@ -27,8 +27,14 @@ class LineItem < FlexCommerceApi::ApiBase
     has_one :item
     belongs_to :cart, class_name: "::FlexCommerce::Cart"
 
-
     # note: only embedded in order responses, not carts
     has_many :line_item_discounts, class_name: "::FlexCommerce::LineItemDiscount"
+
+    # This allows us to set the `path` parameter automatically from the provided `container_id`
+    def self.path(params = nil, *args)
+      params["path"] = { "cart_id" => params["container_id"] } if params["container_id"]
```

Comment:
This is required as we've removed the `_prefix_path` workaround to help with nested / multitenanted line items endpoint ([here](https://github.com/shiftcommerce/flex-ruby-gem/pull/150/files#diff-1fa200ef1a03abce7ff534eaa77f6c15L34)) which causes the upgraded `json_api_client` gem version to error. 

We always have the `container_id` present so this saves us having to pass the `path: { cart_id: ... }` attribute everywhere.

The alternative is we make this a requirement and go through the F/E code adding `path: { cart_id: ... }` everywhere like we've done with nested coupons (and I'm not sure this would work if we're loading line items from a cart, updating them, then re-saving)

Reply:
Another fix would have been to update the platform to always include this path parameter in API responses. Not sure this is a good idea as we'd be adding it specifically for `json_api_client` but it isn't necessarily required for any other json API libraries.